Felienne/hedy
# [BUG FIX] Correctly empty modal prompt on succes

**Description**
In this PR we fix a minor issue where the previous value on the modal prompt was still shown on the inline input. This is fixed by removing the value if the current value is send back to the callback.

**Fixes**
This PR fixes #3246.

**How to test**
Change the password of one of your students. When attempting to change it again verify that the prompt is empty.
